docs/detections/add-exceptions.asciidoc

```diff
@@ -53,7 +53,12 @@ image::images/rule-exception-tab.png[Detail of rule exceptions tab]
 .. Click *Create shared exception list* -> *Create exception item*. 
 --
 
-. In the *Add rule exception* flyout, name the exception and add conditions that define the exception. When the exception's query conditions are met (the query evaluates to `true`), rules do not generate alerts even when other rule criteria are met.
+. In the *Add rule exception* flyout, do the following: 
+
+.. Name the exception. 
+.. Add conditions that define the exception. When the exception's query conditions are met (the query evaluates to `true`), rules do not generate alerts even when other rule criteria are met.
```

Comment:
I had a little difficulty following the second sentence -- there's a lot of passive voice, maybe could be streamlined. I also thought "other rule criteria" sounded a little vague, like maybe some rule criteria are involved while other criteria might not be, but I think it's just the rule itself.
```suggestion
.. Add conditions that define the exception. When the exception's query evaluates to `true`, rules don't generate alerts even when their criteria are met.
```